src/test/java/com/rackspace/salus/monitor_management/services/MonitorManagementTest.java

```diff
@@ -1528,13 +1526,16 @@ public void testRemoveMonitor() {
             .setLabelSelectorMethod(LabelSelectorMethod.AND)
             .setInterval(Duration.ofSeconds(60)));
 
-    final BoundMonitorProjection boundMonitor = new BoundMonitorProjection("e-goner", monitor, "t-1", "r-1", "zoneA");
-
-    when(boundMonitorRepository.findAllByTenantIdAndMonitor_IdIn(anyString(), any()))
-        .thenReturn(Collections.singletonList(boundMonitor));
-
+    BoundMonitor b = new BoundMonitor();
+    b.setMonitor(monitor);
+    b.setResourceId("r-1");
+    b.setEnvoyId("e-goner");
```

Comment:
No need to change, but FYI we have lombok's chainable setters enabled, so you can save a few characters using:
```suggestion
    BoundMonitor b = new BoundMonitor()
      .setMonitor(monitor)
      .setResourceId("r-1")
      .setEnvoyId("e-goner");
```